subscription/scout-details.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Docker Scout subscriptions and features
+description: Learn about the Docker Scout subscriptions plans and features
+keywords: subscription, free, team, business, features
+---
+
+{% include scout-early-access.md %}
+
+This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://docker.com/products/docker-scout){: target="_blank" rel="noopener" class="_" id="dkr_docs_scout_subscription_btl"}.
```

Comment:
**Note for docs reviewer:** what is the id format for these links? I modified a similar one from the subscription page for this but didn't see a process documented for how we come up with these. Would love your input!

Reply:
we're actually a bit naughty and don't use that...When we move to hugo all those jekyll tags will be removed anyway, so no need to worry about them

Reply:
I think we just add them to the Url for now? Like…

```suggestion
This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://docker.com/products/docker-scout?id=dkr_docs_scout_subscription_btl){: target="_blank" rel="noopener" class="_"}.
```

Reply:
What's the ID for? the first example would add an HTML ID attribute to the <a> element. The example @ChrisChinchilla gave would pass the ID as a URL parameter on the HTTP request. There's also a third option of including the ID as an anchor link, and you can do that by simply appending it to the URL:

`https://docker.com/products/docker-scout#dkr_docs_scout_subscription_btl`

But I didn't see any such ID on the target page so I don't think that would be necessary.

Reply:
I assumed it was some form of tracking ID, but I have not seen this one before… Links from Hub and desktop and up with this full string:

https://www.docker.com/products/docker-scout/?utm_source=hub&utm_content=own-tag-details-banner

So maybe

```suggestion
This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://www.docker.com/products/docker-scout/?utm_source=docs){: target="_blank" rel="noopener" class="_"}.
```

?

But perhaps the tag @stephaurelio added has som meaning we're not aware of…

Reply:
I noticed this was how the link to a pricing page was formatted on the core subscription page so was wondering if we need to do these for all links like this but it sounds like maybe that was a one-off and I can remove this?

Reply:
@stephaurelio I am not sure, so if we're not certain and have no explicit instruction, let's remove it then